### DISCUSSION TITLE
Correct 2D Mesh for inserting interface elements in Abaqus

### DISCUSSION THREAD

**Discussion by sylyin0105** [neperfepx/neper#780] [Q&A] [answered]
Hello, and thank you very much for providing the excellent tool NEPER!
I encountered an issue when making a two-dimensional mesh. When I exported it to the Abaqus inp format, I found that the generated mesh did not meet my expectations. I desire a mesh like the one shown in the first diagram, which includes elements of the T3D2 type. However, the generated mesh just consists of elements of the T2D2 type, showed in the second diagram. I want to use this mesh for inserting interface elements between grains by Phon tool.
![屏幕截图 2023-11-29 192127](https://github.com/neperfepx/neper/assets/145850962/6ae7deb3-d8c4-4b31-950a-9c36d04c0d3c)
![屏幕截图 2023-11-29 192158](https://github.com/neperfepx/neper/assets/145850962/79c030c4-91a3-49d2-9627-a50173cd8896)
I would like to know how to adjust the settings to generate the desired mesh. Thank you!

Here's the code I've used.
neper -T -n 10 -dim 2
neper -M n10-id1.tess -dim all -format inp

**Comment by rquey** [comment, marked as answer]
Inserting cohesive elements at interfaces is exactly what `-interface cohesive` does.

**Reply by sylyin0105** [reply]
It really works, thanks for your prompt reply!

**Reply by fine-hhh** [reply]
Hello, I am currently trying to use Phon to insert interface elements. Could you please tell me what the Phon command for this is?
